docs/rst/manual/about/release_notes.rst

```diff
@@ -1,6 +1,75 @@
 Release Notes
 =============
 
+Version 2.10
+------------
+
+Highlighted changes
+~~~~~~~~~~~~~~~~~~~
+
+The shell script jobs will no longer interpret the first path relative to the
```

Comment:
Do we add any info about the fact that it would resolve the path to the datafile first if provided? To be honest, I think it's mostly fruitful to provide information about how it works currently, and what for most users would be the usual case (pretty much as suggested)

Reply:
I'm not sure I follow 🤔 Would you like me to elaborate on the current copy'ing mechanism? I think most users copy from around their configuration file to their runpath and hence will notice the change as the file they're asking to copy not existing anymore. I can of course elaborate on it checking relative to configuration file first, but I don't think that is relevant for most users...

Reply:
> Would you like me to elaborate on the current copy'ing mechanism?

The mechanism prior to this commit was what I hinted to yes. But I agree that we don't need to add it. If the behaviour was stated in the docs on the other hand, then it would make more sense (I believe), and of course we would need to remove them. Now they're pretty much hidden..